### ISSUE TITLE
Missing Buildings in Production Chains

### ISSUE THREAD

**Issue by auddie2015** [NiHoel/Anno1800Calculator#86]
I can't track down what is causing it but something happens when you equip cards into trade halls that give an item bonus which causes many buildings in the production chain to be removed from the screen. This happens upon reaching engineers and the production chains for spectacles and other brass item. Currently, tin and zinc are not showing in the production chain for brass. Gas stations for farms with tractors are reduced to zero needed. What I am not sure is if the calculator is showing a factory that is not actually needed which means that the production chain is not needed.

I hope I explained this properly. If not, I can send screen shots.

**Comment by NiHoel** [comment]
Yes, some items change input or produce extra goods. This might reduce the demand of some goods to zero. Buildings not needed are not shown.

If you want a detailed explanation of have the feeling that something isn't correct, send me the configuration of the calculator (download button top right)

**Comment by auddie2015** [comment]
I will double check my equipped cards and things checked off in the calculator. This late in the game there are so many things going on that it gets confusingly complex even with a calculator. I may need an AI to replace me in the production chain instead!

Thank you for your awesome tool!
